core/jazz_services/components/utils.js

```diff
@@ -162,7 +163,26 @@ var getUpdateData = function (update_data) {
             } else {
                 input_data[field] = [];
             }
-        } else {
+        }
+        else if (update_data[field] && update_data[field].constructor === Object) {
+            var ref_data = old_data.validateServiceExists['service_payload'];
+            var new_keys = Object.keys(update_data[field]);
+            var old_keys = Object.keys(ref_data[field]);
+            var update_object = update_data[field];
+            var new_object = ref_data[field];
+            if (new_keys.length > 0) {
+                for (var i = 0; i < new_keys.length; i++) {
```

Comment:
use let

Reply:
should i have to update whole folder's var or mine only ??

Reply:
and i run prettier to fix indentation